src/components/spreadsheet/data-fetching/use-load-equipment.ts

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import { useCallback, useEffect, useMemo, useState } from 'react';
+import { SpreadsheetEquipmentsByNodes, SpreadsheetEquipmentType } from '../config/spreadsheet.type';
+import { UUID } from 'crypto';
+import { formatFetchedEquipments } from '../utils/equipment-table-utils';
+import { useDispatch, useSelector } from 'react-redux';
+import { NodeType } from '../../graph/tree-node.type';
+import { isStatusBuilt } from '../../graph/util/model-functions';
+import { AppState } from '../../../redux/reducer';
+import { loadEquipments } from '../../../redux/actions';
+import { getFetcher } from './fetchers';
+
+export const useLoadEquipment = (
+    type: SpreadsheetEquipmentType,
+    studyUuid: UUID | null,
+    currentRootNetworkUuid: UUID | null,
+    currentNodeUuid: UUID | undefined,
```

Comment:
This shouldn't be passed, it should use useSelector in the hook directly

Reply:
yes, done

Reply:
hook deleted in the meantime